web-dapp/logger.js

```diff
@@ -1,12 +0,0 @@
-'use strict';
```

Comment:
Let's move everything from `utils` to `server-lib`, no need to have a new folder